packages/now-node/src/helpers.ts

```diff
@@ -49,16 +49,8 @@ function getBodyParser(req: NowRequest, body: Buffer) {
 function getQueryParser({ url = '/' }: NowRequest) {
   return function parseQuery(): NowRequestQuery {
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const { URL } = require('url');
-    // we provide a placeholder base url because we only want searchParams
-    const params = new URL(url, 'https://n').searchParams;
-
-    const query: { [key: string]: string | string[] } = {};
-    for (const [key, value] of params) {
-      query[key] = value;
-    }
-
-    return query;
+    const { parse: parseURL } = require('url');
```

Comment:
FYI, you can achieve the same behavior with `url.searchParams.getAll('a')` which returns an array.

Although I think the solution in this PR is more elegant and probably less error-prone. 

cc @TooTallNate @lucleray

Reply:
Arguably it would have been better to just return `url.searchParams` and allow `req.query.get('foo')` work as expected, but that ship has sailed.

Reply:
The goal was to stay as close as possible to express (https://expressjs.com/fr/4x/api.html#req.query).